training/code-of-conduct/training-privacy.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: section
+title: ARCHER2 Training Privacy Statement
+banner: web_banners_05.jpg
+---
+
+
+The information you provide will be used by EPCC to administer your attendance at our training courses. The information will be used to send you information about the course and will be retained beyond the end of the course so that we can send you follow-up surveys.
+
+We are using this information about you because it is necessary for the performance of the contracts that provide you with these training courses.
+
+Information about you will be shared with:
+
+- The organisations that host the training courses that you attend. This is necessary to allow them to ensure the courses run smoothly.
+- External bodies (such as UKRI, EPSRC, Software Carpentry, PRACE, ...) that fund or support the courses you attend. This is necessary to justify their funding or support for these courses.
+
+We will retain the personal data you provide for a period of up to 2 years. Any data held beyond this point will be anonymised.
+
+If you have any questions, please contact our training team at training@epcc.ed.ac.uk
```

Comment:
Can we make the e-mail address a link? [training@epcc.ed.ac.uk](mailto:training@epcc.ed.ac.uk)